starters/basic-starter/pages/[...slug].tsx

```diff
@@ -21,16 +25,57 @@ const CONTENT_TYPES = [
   'node--event',
   'node--person',
   'node--place',
+  'taxonomy_term--article_type',
+  'taxonomy_term--categories',
+  'taxonomy_term--event_type',
+  'taxonomy_term--person_type',
+  'taxonomy_term--place_type',
 ];
 
 interface NodePageProps extends LayoutProps {
-  node: DrupalNode;
+  // getStaticProps() will return an array as the node when a user requests /taxonomy/term/3 for example.
+  node: DrupalNode | DrupalNode[];
+  label?: string;
+  type?: string;
 }
 
-export default function NodePage({ node, menus }: NodePageProps) {
+export default function NodePage({ node, menus, label, type }: NodePageProps) {
   if (!node) return null;
 
-  return (
+  return Array.isArray(node) ? (
+    <Layout title={label} menus={menus}>
+      {type === 'taxonomy_term--article_type' &&
```

Comment:
I am checking the `type` so existing taxonomy terms that don't have any nodes will still render a page with the term shown on the page